crates/prelude/src/button.rs

```diff
@@ -64,15 +65,16 @@ impl<H: Handler> Listener<H> {
     /// # Examples
     ///
     /// ```ignore
-    /// Listener::new(index, |state| debug!("Button has been {state:?}"))
+    /// Listener::new(index, |state| debug!("Button has been {state:?}"))?
     /// ```
-    pub fn new(button: usize, handler: H) -> Self {
+    #[track_caller]
```

Comment:
No need for track-caller if it never panics.

Reply:
Removed.